hedyorg/hedy
# 💻 format all old adventure contents with curlies

Fixes #5310 

**How to test?**
- have an existing adventure with curly braces
- try to see this adventure from different locations; /hedy, /public-adventures, etc.
